neverbleed.c

```diff
@@ -505,10 +527,28 @@ void neverbleed_transaction_write(neverbleed_t *nb, neverbleed_iobuf_t *buf)
     iobuf_transaction_write(buf, thdata);
 }
 
+static int exdata_idx = -1;
+static void exdata_free_callback(void *parent, void *ptr, CRYPTO_EX_DATA *ad, int idx, long argl, void *argp)
+{
+    if (idx != exdata_idx) return;
+    if (ptr == NULL) return;
```

Comment:
Do we have a case where ptr is going to be NULL? Assuming that there is none, maybe `assert` would be a better choice?